src/Files.App/Actions/Content/PlayAllAction.cs

```diff
@@ -1,30 +1,26 @@
 ﻿// Copyright (c) 2023 Files Community
 // Licensed under the MIT License. See the LICENSE.
 
-using CommunityToolkit.Mvvm.ComponentModel;
-using CommunityToolkit.Mvvm.DependencyInjection;
 using Files.App.Commands;
 using Files.App.Contexts;
-using Files.App.Extensions;
-using Files.App.Helpers;
-using Files.Backend.Helpers;
-using System.ComponentModel;
-using System.Linq;
-using System.Threading.Tasks;
 
 namespace Files.App.Actions
 {
 	internal class PlayAllAction : ObservableObject, IAction
 	{
 		private readonly IContentPageContext context = Ioc.Default.GetRequiredService<IContentPageContext>();
 
-		public string Label { get; } = "PlayAll".GetLocalizedResource();
+		public string Label
+			{ get; } = "PlayAll".GetLocalizedResource();
 
-		public string Description { get; } = "PlayAllDescription".GetLocalizedResource();
+		public string Description
+			{ get; } = "PlayAllDescription".GetLocalizedResource();
 
-		public RichGlyph Glyph { get; } = new("\uE768");
+		public RichGlyph Glyph
+			{ get; } = new("\uE768");
```

Comment:
Shouldn't you replace `{ get; }` even here?

Reply:
Fixed